src/components/SidebarItem/_sidebar-item.scss

```diff
@@ -0,0 +1,87 @@
+@import '../../globals/scss/vars';
+@import '../../globals/scss/colors';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/spacing';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/css--reset';
+
+@include exports('sidebar-item') {
+  .#{$prefix}--sidebar-item-wrapper {
+    display: flex;
+    flex-direction: row;
+    background: var(--ui-01);
+    height: 95px;
+    padding: $spacing-md;
+    // border-bottom: 1px solid var(--ui-02);
+    cursor: pointer;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.15);
+    justify-content: space-between;
+
+    .#{$prefix}--sidebar-item__content {
+      display: flex;
+      flex-direction: row;
+    }
+
+    .#{$prefix}--profile-image {
+      width: rem(60px);
+      height: rem(60px);
+      border-radius: 50%;
+      overflow: hidden;
+      background-color: var(--ui-02);
+      margin-right: $spacing-md;
+
+      > img {
+        object-fit: cover;
+        width: 100%;
+        height: 100%;
+      }
+
+      .#{$prefix}--profile-image--empty {
+        width: 100%;
+        height: 100%;
+        border-radius: 50%;
+        background-size: 100%;
+        border: 2px solid white;
+        background-color: $color__navy-gray-7;
+        text {
+          fill: var(--ui-01);
+          font-weight: 600;
+          font-family: 'Arial';
+        }
+      }
+    }
+
+    .#{$prefix}--sidebar-content-wrapper {
+      flex-direction: column;
+      margin-right: 0.5rem;
+      line-height: 22px;
+
+      .#{$prefix}--item-title {
+        font-weight: 600;
+        font-size: 16px;
```

Comment:
Avoid using px as sizes. Use `em` or `rem` instead. There are also variables available for different spacings (`$spacing-xs`, etc.)
https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/

Reply:
sure,Thanks for drawing my attention to it